src/components/menu/examples/menu-grid.tsx

```diff
@@ -3,15 +3,16 @@ import { Component, h } from '@stencil/core';
 
 /**
  * With grid layout
- * To render items of a menu in a gird layout instead of a vertical list,
- * two things are required:
- * 1. Simply setting the `gridLayout` property to `true`, and
- * 1. Specifying a proper width for the menu surface, using the
- * `--menu-surface-width` variable.
+ * To render items of a menu in a grid layout instead of a vertical list,
+ * simply setting the `gridLayout` property to `true`.
  *
  * :::note
- * Do not forget that menus should behave responsively, thus using a fixed `width`
- * should be avoided. To make the width responsive, try using the `min()` function.
+ * The menu surface of menus with the grid layout has a responsive width by default,
+ * which will not grow wider than a certain size. However, if the default size is not
+ * wide enough for your use case, you can try setting another responsive width, using
+ * the `--menu-surface-width` variable.
```

Comment:
I think it's a pity that we've gone with `--menu-surface-width`. I feel like `--menu-max-width` had been more concise and to the point. Perhaps we should deprecate `--menu-surface-width` in favour of `--menu-max-width`, but we can just as well do that in a separate PR…

When naming things that is seen by the consumer, please try to take one step back, and look at what the consumer, or the developer of that consumer, sees. If in doubt, please do reach out for a second opinion 😊

Reply:
(I know the name was already used before this PR, just mentioning it here, since I commented about not using "the surface of menus" above.)